modules/openapi-generator/src/test/java/org/openapitools/codegen/serializer/SerializerUtilsTest.java

```diff
@@ -178,7 +178,7 @@ public void testToYamlStringMinimalExample() throws Exception {
                 "      description: Some description\n" +
                 "      operationId: pingOp\n" +
                 "      responses:\n" +
-                "        200:\n" +
+                "        \"200\":\n" +
```

Comment:
@OpenAPITools/generator-core-team HTTP response code is now in the form of string instead of integer.

I don't see any mention of this change in swagger core/parser change log or release note.

I think we should highlight this change in our release note so users won't be surprised if the HTTP status code is now a string instead of a integer.

Reply:
Can't we convert it before consuming?

Reply:
We can but someone may have already processed these numeric status codes in their app and this change may break their app. To fix it, the users need to do the conversion.

Reply:
@wing328 this is documented on the Responses object:

> This field MUST be enclosed in quotation marks (for example, "200") for compatibility between JSON and YAML.

Reply:
I believe YAML keys are always considered strings, so someone parsing this manually through YAML should be unaffected. We could verify in YAML spec, but probably lower priority than fixing a bug that makes the doc non-compliant with the spec.

Reply:
OK. Let's go ahead with this change.